tests/Unit/DefaultValuesTest.php

```diff
@@ -349,7 +346,7 @@ public function default_takes_precedence_over_nullable_or_empty_array(): void
             ['one' => new Property($this->factory, 'one', ['null', 'array'], [], true, 'blim')],
             DataTransferObject::class,
             [],
-            NULLABLE_DEFAULT_TO_NULL | ARRAY_DEFAULT_TO_EMPTY_ARRAY | DEFAULTS
+            DEFAULTS
```

Comment:
Do we think DEFAULT_VALUES might be a better constant name - DEFAULTS is nice and terse but maybe not explicit enough.

Reply:
Will change to WITH_DEFAULTS after merging to avoid conflicts.